components/ws-manager/pkg/manager/monitor.go

```diff
@@ -142,15 +142,13 @@ func (m *Monitor) Start() error {
 		log.WithError(err).Warn("cannot mark all existing workspaces active - this will wrongly time out user's workspaces")
 	}
 
-	return nil
-}
+	go func() {
+		for range m.ticker.C {
+			m.doHousekeeping(context.Background())
```

Comment:
before we'd start on go-routine on every ticker. Is the change intended?

Reply:
Indeed, it is. This way we don't run multiple concurrent housekeeping operations at the same time.
[Ticker adjusts its interval/drops ticks to make up for slow receivers.](https://golang.org/pkg/time/#Ticker)

Reply:
> Indeed, it is.

:+1: 

> Ticker adjusts its interval/drops ticks to make up for slow receivers.

That's nice, didn't know that!